src/components/middle/message/Message.tsx

```diff
@@ -850,8 +850,21 @@ const Message: FC<OwnProps & StateProps> = ({
       animateUnreadReaction({ messageIds: [messageId] });
     }
 
+    const unreadMentionsIds: number[] = [];
     if (message.hasUnreadMention) {
-      markMentionsRead({ messageIds: [messageId] });
+      unreadMentionsIds.push(message.id);
+    }
+
+    if (album) {
+      for (const albumMessage of album.messages) {
+        if (albumMessage.hasUnreadMention) {
+          unreadMentionsIds.push(albumMessage.id);
```

Comment:
That would duplicate the main message id in albums. You could make `unreadMentionIds` re-assignable and write it like
```ts
if (album) {
  unreadMentionIds = album.messages.filter((msg) => msg.hasUnreadMention).map((msg) => msg.id);
}
```

Reply:
Sorry for the delay. I made an improvement.